types/router.go

```diff
@@ -55,7 +55,7 @@ func (r Route) Handler() Handler {
 
 // Empty returns true only if both handler and path are not empty.
 func (r Route) Empty() bool {
-	return r.handler == nil || r.path == strings.TrimSpace("")
```

Comment:
`strings.TrimSpace("")` doesn't make any sense. It should be on `path`. I move trimming to line 43.

Reply:
Makes sense